backend/core/views.py

```
@@ -4895,3 +4895,65 @@ class FindingViewSet(BaseModelViewSet):
     @action(detail=False, name="Get status choices")
     def status(self, request):
         return Response(dict(Finding.Status.choices))
+
+
+class IncidentViewSet(BaseModelViewSet):
+    model = Incident
```

Ruff (0.8.2)

`Incident` may be undefined, or defined from star imports

(F405)